content/docs/1.6.0/monitoring/prometheus-and-grafana-setup.md

```diff
@@ -40,6 +40,8 @@ Follow instructions in [Prometheus Operator - Quickstart](https://github.com/pro
 
 ### Install Longhorn ServiceMonitor
 
+#### Install Longhorn ServiceMonitor with Kubectl
+
 1. Create a ServiceMonitor for Longhorn manager.
```

Comment:
```suggestion
Create a ServiceMonitor for Longhorn Manager.
```

Reply:
There is only one step so we should not use an ordered list.

Reply:
Modified. Thanks